check.py

```diff
@@ -301,23 +250,28 @@ def minify_check(wast, verify_final_result=True):
   assert e in err, 'Expected help to contain program name, got:\n%s' % err
   assert len(err.split('\n')) > 8, 'Expected some help, got:\n%s' % err
 
-print '\n[ checking binaryen-shell -o notation... ]\n'
+print '\n[ checking wasm-opt -o notation... ]\n'
 
 wast = os.path.join('test', 'hello_world.wast')
 delete_from_orbit('a.wast')
-cmd = [os.path.join('bin', 'binaryen-shell'), wast, '-o', 'a.wast']
+cmd = [os.path.join('bin', 'wasm-opt'), wast, '-o', 'a.wast']
 run_command(cmd)
 fail_if_not_identical(open('a.wast').read(), open(wast).read())
 
-print '\n[ checking binaryen-shell passes... ]\n'
+print '\n[ checking wasm-opt passes... ]\n'
 
 for t in sorted(os.listdir(os.path.join('test', 'passes'))):
   if t.endswith('.wast'):
     print '..', t
     passname = os.path.basename(t).replace('.wast', '')
     opts = ['-O'] if passname == 'O' else ['--' + p for p in passname.split('_')]
-    cmd = [os.path.join('bin', 'binaryen-shell')] + opts + [os.path.join('test', 'passes', t), '--print']
-    actual = run_command(cmd)
+    t = os.path.join('test', 'passes', t)
+    actual = ''
+    for module, asserts in split_wast(t):
+      assert len(asserts) == 0
+      open('split.wast', 'w').write(module)
```

Comment:
Maybe we should close this handle at the end of the iteration.
or just put it in a `with open` clause.